en/docs/tutorials/create-your-first-scheduled-task.md

```diff
@@ -0,0 +1,218 @@
+        | **Variable Name**    | `payload`                        |
+        | **Value Expression** | `getPopulationByCountryResponse` |
+   
+        The expression refers to the `getPopulationByCountryResponse` variable from the World Bank connector.
+
+    3. Click **Save**.
+
+6. Now let's add a log statement to log the payload generated via the last variable as follows:
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Click **Log** to add a log statement.
+   
+    3. In the **Expression** field, enter `payload.toString()`. This expression gets the value of the `payload` variable statement you added, converts it to a string format, and logs it.
+   
+    4. Click **Save**.
+
+Now you have completed designing the functionality of the scheduled task.
+
+The low-code diagram looks as follows:
+
+![Scheduled task low-code view](../assets/img/tutorials/scheduled-task/scheduled-task-low-code-view.png){.cInlineImage-half}
+
+The pro-code view looks as follows:
+
+!!! tip
+    To access the pro-code view, click the **<**/**>** icon.
+
+```
+import ballerina/log;
+import ballerinax/worldbank;
+
+public function main() returns error? {
+
+    worldbank:Client worldbankEndpoint = check new ({});
+    worldbank:IndicatorInformation[] getPopulationByCountryResponse = check worldbankEndpoint->getPopulationByCountry("USA");
+    anydata payload = getPopulationByCountryResponse;
+    log:printInfo((payload).toString());
+}
+```
+
+### Step 1.3: Run the scheduled task
+
+To check whether the scheduled task is executable, click **Run**.
+
+![Run scheduled task](../assets/img/tutorials/scheduled-task/run-scheduled-task.png){.cInlineImage-half}
+
+If the scheduled task configuration is free of errors, the following appears in the terminal log.
+
+ ```
+ Running executable
+ ```
+
+### Step 1.4: Sync changes with Choreo Console
+
+The scheduled task configuration you created is currently only saved in the Web Editor. To deploy it and run it, you need to push this configuration to the Choreo Console by saving it in a GitHub repository. To do this, follow this procedure:
+
+!!! info
+    You are saving the scheduled task in a Choreo-maintained repository as specified in [Step 1.1: Create a new project and add a scheduled task component](#step-11-create-a-new-project-and-add-a-scheduled-task-component).
+
+1. Click **Sync with Choreo Upstream** in the bottom panel of the page (highlighted in red). In the message that appears, click **Sync my changes with Choreo**.
+
+    ![Sync with Choreo upstreeam](../assets/img/tutorials/scheduled-task/sync-with-choreo-upstream.png){.cInlineImage-full}
+
+2. In the left panel, enter a commit message (e.g., `Implement Scheduled Task`) and click on the tick.
+
+    ![Commit message](../assets/img/tutorials/scheduled-task/commit-message.png){.cInlineImage-full}
+
+    Select **Yes** in the message that appears to specify that you need to stage the changes.
```

Comment:
Shall we say "Click **Yes** in the message that appears . This will stage your changes."